### DISCUSSION TITLE
INSTALLAZIONE REGISTRO ELETTRONICO "GIUA SCHOOL" GRATUITAMENTE

### DISCUSSION THREAD

**Discussion by registro-giuaschool** [iisgiua/giuaschool#726] [Idee e proposte]
Buongiorno a tutti, se i responsabili dell'applicativo giua@school, sono d'accordo, potete contattarmi all'indirizzo mail: registroelettronicogiuaschool@gmail.com e vi fornirò una demo e in più se volete posso installarvi gratuitamente il registro elettronico personalizzato.

--

Cordiali Saluti 
Accetto le e-mail da tutti

**Comment by trinko** [comment]
Ottima iniziativa. Grazie.

**Comment by murfivladi** [comment]
Ciao
